java/aquality-selenium-template/src/test/java/aquality/selenium/template/glue/hooks/BrowserHooks.java

```diff
@@ -0,0 +1,18 @@
+package aquality.selenium.template.glue.hooks;
+
+import aquality.selenium.browser.AqualityServices;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+
+public class BrowserHooks {
+
+    @Before
+    public void startBrowser() {
+        AqualityServices.getBrowser();
```

Comment:
I think we could remove this hook, because Equality will create Browser on first call automatically and if user will have e.g. Api tests (where browser is not necessary), this structure could force him to find some workarounds